lib/themes/dosomething/paraneue_dosomething/js/components/Slideshow.js

```diff
@@ -21,6 +22,7 @@ class Slideshow extends React.Component {
 
   moveBackward() {
     const slide = this.state.currentSlide - 1;
+    sendEvent('Onboarding', 'Slide Change', slide);
```

Comment:
Is it worth making this 'Slide Change (Next button)" ? I should ask a data person whats more valuable I guess